src/main/resources/db/migration/common/V1_1__move-to-cell.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE "move_to_cell"
```

Comment:
Can we call this eg cell_movement? It won't always be a movement _to_ a cell, it could just as well be unassigning a cell